acquia/cli
# Fix #300: Download Drush 9 aliases

Motivation
----------
Fixes #300 

Proposed changes
---------
Instead of downloading Drush 8 aliases, downloading Drush 9 ones and install them in `~/.drush/sites` instead of `~/.drush` per conventions.

Alternatives considered
---------
Offer the user the option of downloading Drush 8 or 9 aliases, or download and install both (sequentially). I don't know how many people still use Drush 8 and would want aliases for it. @grasmash @anavarre opinions?

Testing steps
---------
Run `acli remote:aliases:download` and ensure aliases are populated at `~/.drush/sites`.
